extensions/agroal/deployment/src/test/resources/application-default-datasource.properties

```diff
@@ -1,16 +1,17 @@
 #tag::basic[]
-quarkus.datasource.url=jdbc:h2:tcp://localhost/mem:default
-quarkus.datasource.driver=org.h2.Driver
+quarkus.datasource.kind=h2
```

Comment:
I prefer `system` or `db-system` (or maybe `product`) than `kind` see https://groups.google.com/forum/#!searchin/quarkus-dev/kind%7Csort:date/quarkus-dev/zos4bOUVNHY/tYcS7IwCBAAJ

CC @Sanne

Reply:
Can't say like it. @Sanne WDYT?